changelog.d/11633.misc

```diff
@@ -0,0 +1,2 @@
+Stop building bionic, update dh-virtualenv and update Trove classifiers, tasks all associated with dropping
+Python 3.6, which is EOL.
```

Comment:
I think these need to be on one line. In any case I'd probably just say "Drop support for Python 3.6 and Ubuntu 18.04."